modules/registry-third-party-registries.adoc

```diff
@@ -20,3 +20,24 @@ deployment reactions occur.
 {product-title} can communicate with registries to access private image
 repositories using credentials supplied by the user. This allows {product-title}
 to push and pull images to and from private repositories.
+
+[id="registry-authentication_{context}"]
+=== Registry authentication with Podman
```

Comment:
This part has duplicate content with https://docs.openshift.com/container-platform/4.6/registry/index.html#registry-authentication-enabled-registry-overview_registry-overview , could we look into it?

Reply:
If that's the case then I think this PR is not required.